src/IQFeed.CSharpApiClient/Streaming/Level1/Level1Client.cs

```diff
@@ -1,14 +1,13 @@
 ﻿using System;
 using System.Threading.Tasks;
-using IQFeed.CSharpApiClient.Common;
 using IQFeed.CSharpApiClient.Socket;
 using IQFeed.CSharpApiClient.Streaming.Common.Messages;
 using IQFeed.CSharpApiClient.Streaming.Level1.Handlers;
 using IQFeed.CSharpApiClient.Streaming.Level1.Messages;
 
 namespace IQFeed.CSharpApiClient.Streaming.Level1
 {
-    public class Level1Client : ILevel1Client
+    public class Level1Client : Level1ClientBase, ILevel1Client
```

Comment:
I'll update sorry but should be BaseLevel1Client.

Reply:
fixed